src/external/atomsbase.jl

```diff
@@ -48,6 +48,10 @@ function parse_system(system::AbstractSystem{D}) where {D}
         error("Charged systems not yet supported in DFTK.")
     end
 
+    if !all([atom.atomic_mass for atom in system] .== atomic_mass.(atoms))
```

Comment:
This should work, I believe:
```suggestion
    if atomic_mass(system) != atomic_mass.(atoms))
```

Reply:
They simply test different things (one is the annotated charge per atom, one the annotated total charge of the system ... sometimes you want one, sometimes the other ... don't ask, it's a mess). We can also have a tolerance in the second test if that puts you more at ease.